phpunit.xml.dist

```diff
@@ -16,6 +16,7 @@
     <php>
         <!-- https://github.com/symfony/console/blob/v3.3.12/Terminal.php#L24-L36 -->
         <env name="COLUMNS" value="120"/>
+        <server name="CFP_ENV" value="testing"/>
         <ini name="memory_limit" value="512M"/>
```

Comment:
Can we keep these ordered alphabetically?

This has the advantage that it will be more obvious where to add new configuration items here.

Reply:
Order the variable names? Or the XML element names?

```xml
<env name="COLUMNS" value="120"/>
<ini name="memory_limit" value="512M"/>
<server name="CFP_ENV" value="testing"/>
```

vs

```xml
<server name="CFP_ENV" value="testing"/>
<env name="COLUMNS" value="120"/>
<ini name="memory_limit" value="512M"/>
```

Reply:
I'd suggest to order the element names, or even better, just use the https://plugins.jetbrains.com/plugin/5919-lines-sorter plugin for PHPStorm (if you use it), and then press <kbd>Option</kbd> + <kbd>Shift</kbd> + <kbd>L</kbd>.